packages/mdc-textfield/_mixins.scss

```diff
@@ -63,36 +63,79 @@
   }
 }
 
+@mixin mdc-text-field-bottom-line-color($color) {
+  &:not(.mdc-text-field--disabled) {
+    @include mdc-text-field-bottom-line-color_($color);
+  }
+}
+
+@mixin mdc-text-field-bottom-line-hover-color($color) {
```

Comment:
`mdc-text-field-hover-bottom-line-color` is better I think. Since you hover on the text field, not the bottom line.